# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rt2500/dist.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- rt2500-1.1.0-b4/Module/Makefile.orig	2006-08-23 12:22:03.000000000 +0200
+++ rt2500-1.1.0-b4/Module/Makefile	2006-08-23 12:22:49.000000000 +0200
@@ -116,25 +116,11 @@
 	@rm -fr .tmp_versions
 
 modules_install:
-ifeq ($(PATCHLEVEL),4)
-	if ! [ -f $(MODULE_OBJECT) ]; then \
-	module; \
-	fi
 	@echo "install '$(MODULE_OBJECT)' to $(MODULE_ROOT)"
 	install -m 755 -o 0 -g 0 -d $(MODULE_ROOT)
 	install -m 644 -o 0 -g 0 $(MODULE_OBJECT) $(MODULE_ROOT)
-	/sbin/depmod -a
-else
-	echo "2.6 module install"
-	make $(KBUILD_PARAMS) modules_install
-	/sbin/depmod -a
-endif
 
 install: modules_install
-	@if ! grep -q 'ra0' $(MODULE_CONF) ; then \
-		echo "append 'alias ra0 rt2500' to $(MODULE_CONF)"; \
-		echo "alias ra0 rt2500" >> $(MODULE_CONF) ; \
-	fi
 
 install-fedora:
 	if ! [ -f $(MODULE_OBJECT) ]; then \
